dspace-api/src/main/java/org/dspace/content/FeedbackServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content;
+import java.io.IOException;
+import java.util.Date;
+import java.util.Objects;
+import javax.mail.MessagingException;
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.commons.lang.StringUtils;
+import org.dspace.content.service.FeedbackService;
+import org.dspace.core.Context;
+import org.dspace.core.Email;
+import org.dspace.core.I18nUtil;
+
+public class FeedbackServiceImpl implements FeedbackService {
+
+    @Override
+    public void sendEmail(Context context, HttpServletRequest request, String recipientEmail, String senderEmail,
+            String message, String page) throws IOException, MessagingException {
+        String session = request.getSession().getId();
```

Comment:
The backend is supposed to be stateless, which means we don't / shouldn't have a Session (or a Session ID) on the backend.  I'd recommend we replace this with the `correlationID`, as that's the "session-like" ID which we use in our dspace.log.